submissions/chri911/popup-menu/index.html

```diff
@@ -0,0 +1,106 @@
+                    <img src="images/contacts.png" alt="">
+                    <div class="text">Contacts</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/tanslation.png" alt="">
+                    <div class="text">Translate</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/videos.png" alt="">
+                    <div class="text">Videos</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/search.png" alt="">
+                    <div class="text">Search</div>
+                </a>
+                <a href="#" class="item photo">
+                    <img src="images/photos.png" alt="">
+                    <div class="text">Photo</div>
+                </a>
+                <a href="#" class="item apps">
+                    <img src="images/apps.png" alt="">
+                    <div class="text">Apps</div>
+                </a>
+                <a href="#" class="item photo">
+                    <img src="images/documents.png" alt="">
+                    <div class="text">Document</div>
+                </a>
+            </div>
+                <div class="more-btn1">
+                    <label for="expanded" id="more-btn">More</label>               
+               </div>
+                    <div class="more__items">
+                        <a href="#" class="item">
+                            <img src="images/chat.png" alt="">
+                            <div class="text">Chat</div>
+                        </a>
+                        <a href="#" class="item">
+                            <img src="images/classroom.png" alt="">
+                            <div class="text">Classroom</div>
+                        </a>
+                        <a href="#" class="item collection">
+                           <img src="images/collection.png" alt="">
+                            <div class="text">Collection</div>
+                        </a>
+                        <a href="#" class="item ring">
+                            <img src="images/mars.png" alt="">
+                            <div class="text">Mars</div>
+                        </a>
+                        <a href="#" class="item ring">
+                            <img src="images/blogs.png" alt="">
+                            <div class="text">Blogs</div>
+                        </a>
+                        <a href="#" class="item ring">
+                            <img src="images/ads.png" alt="">
+                            <div class="text">Ads</div>
+                        </a>
+                    </div>
+                    <a href="#" class="more-btn">Even more from Kottans</a>
+                    </div>
+                    </div>
+             </div>        
+        <div class="notes"></div>
+        <img class="user"></img>
+    </div>
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)
You can automate this by turn on this option in your code redactor.